src/marks/contour.js

```diff
@@ -115,7 +115,8 @@ function contourGeometry({thresholds, interval, ...options}) {
     const {pixelSize: k, width: w = Math.round(Math.abs(dx) / k), height: h = Math.round(Math.abs(dy) / k)} = this;
     const kx = w / dx;
     const ky = h / dy;
-    const V = channels.value.value;
+    const temporal = isTemporal(channels.value.value);
+    const V = temporal && this.blur > 0 ? Float64Array.from(channels.value.value) : channels.value.value;
```

Comment:
Do we want to use `coerceNumbers` here whenever `this.blur > 0`, not just in the `temporal` case? I also wonder if we could provide a hint earlier when the _value_ channel is declared to say that it must include numbers (at least in the `this.blur > 0` case).

Reply:
I'd love to make contours work with categorical data, but I feel it would necessitate a completely different approach (we'd have to compute the contours for each of the categories on a boolean raster saying whether a pixel belongs to that category… something like that… with blurring to accomodate the "random-walk" interpolator).